_config/config.yml

```diff
@@ -1,5 +1,32 @@
 ---
-Name: graphql-devtool-routes
+Name: graphql-devtools
+After:
+  - 'graphql-dev'
+---
+SilverStripe\GraphQLDevTools\Controller:
+  default_route: 'graphql'
+
+
+---
+Name: graphql4-devtool-routes
+After:
+  - 'graphql-dev'
+Only:
+  classexists: 'SilverStripe\GraphQL\Schema\Schema'
+---
+SilverStripe\GraphQL\Dev\DevelopmentAdmin:
+  registered_controllers:
+    ide:
+      controller: 'SilverStripe\GraphQLDevTools\Controller'
+      links:
+        ide: Run the GraphQL IDE
+
+---
+Name: graphql3-devtool-routes
+After:
+  - 'graphql-dev'
+Except:
+  classexists: 'SilverStripe\GraphQL\Schema\Schema'
```

Comment:
Doesn't actually work with GraphQL 3.x because you've removed `SilverStripe\GraphQLDevTools\GraphiQLController'`

Reply:
yup, caught that one too